client/src/features/projectsV2/new/ProjectV2New.tsx

```diff
@@ -111,6 +111,10 @@ export default function ProjectV2New() {
 function ProjectV2CreationDetails() {
   const [createProject, result] = usePostProjectsMutation();
   const navigate = useNavigate();
+  const groupMatch = useMatch(ABSOLUTE_ROUTES.v2.groups.show.root);
+  const groupSettingsMatch = useMatch(ABSOLUTE_ROUTES.v2.groups.show.settings);
```

Comment:
You can replace this with a splat route, it will match any group route, even if we change them in the future:
```suggestion
  const groupMatch = useMatch(`${ABSOLUTE_ROUTES.v2.groups.show.root}/*`);
```

We could also add: `ABSOLUTE_ROUTES.v2.groups.show.splat = "/v2/groups/:slug/*"` as a constant.

See: https://reactrouter.com/start/declarative/routing#splats

Reply:
This is a good suggestion. I have implemented it this way in 437f8523daad1715ede914b7806e198cd7b10a58.